requirements/js_test.txt

```diff
@@ -5,9 +5,9 @@
 #    pip-compile --output-file requirements/js_test.txt requirements/js_test.in
 #
 argparse==1.4.0           # via jasmine
-backports.functools-lru-cache==1.5  # via cheroot, jaraco.functools
+backports.functools-lru-cache==1.5  # via cheroot
 cheroot==6.5.2            # via cherrypy
-cherrypy==17.3.0          # via jasmine
```

Comment:
@georgebabey: You can see here that cherrypy used to be a dependency that just came from jasmine.  When I ran `make upgrade`, this continued to be true, but cherrypy was also upgraded but that Jasmine tests started failing in Travis due to the upgrade.  So, I temporarily pinned it so Jasmine tests would continue to pass.